data/trie/evictionWaitingList/evictionWaitingList.go

```diff
@@ -0,0 +1,91 @@
+package evictionWaitingList
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data/trie"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+// evictionWaitingList is a structure that caches keys that need to be removed from a certain database.
```

Comment:
add to this description that the structure is not concurrent safe but it will be used in the trie where all accesses are protected by a mutex

Reply:
done